test/torchaudio_unittest/models/wav2vec2/huggingface_intergration_test.py

```diff
@@ -213,6 +310,15 @@ def test_recreate_pretrain(self, config, factory_func):
         reloaded.eval()
         self._test_recreate(imported, reloaded, config)
 
+    @WAVLM_CONFIGS
+    def test_recreate_wavlm(self, config, factory_func):
```

Comment:
can this function be merged with `test_recreate_wavlm` above, as there is no finetune/aux case in wavlm model?